website/content/en/preview/AWS/provisioning.md

```diff
@@ -85,7 +85,7 @@ To verify if this restriction affects you, run the following commands.
 ```bash
 CLUSTER_VPC_ID="$(aws eks describe-cluster --name $CLUSTER_NAME --query cluster.resourcesVpcConfig.vpcId --output text)"
 
-aws ec2 describe-security-groups --filters Name=vpc-id,Values=$CLUSTER_VPC_ID Name=tag-key,Values=kubernetes.io/cluster/$CLUSTER_NAME --query SecurityGroups[].[GroupName] --output text
+aws ec2 describe-security-groups --filters Name=vpc-id,Values=$CLUSTER_VPC_ID Name=tag-key,Values=kubernetes.io/cluster/$CLUSTER_NAME --query 'SecurityGroups[].[GroupName]' --output text

```

Comment:
Thank you very much for submitting the change.
I ran the two command and I don't see any difference. Is there any reason that the single quotation marks are needed?

Reply:
I’m getting an error on my Mac. I’ll update the OP with the screenshot once I get to my computer.

Reply:
I'm using zsh... that's why. 

![image](https://user-images.githubusercontent.com/31543/153338876-3bbbd860-86d8-4ee6-9452-0802fc381287.png)